app/controllers/projects_controller.rb

```diff
@@ -12,7 +12,10 @@ class ProjectsController < ApplicationController
   def index
     @projects = {}
 
-    projects_joined = policy_scope(Project).preload(:tag_group)
+    projects_joined = policy_scope(Project)
```

Comment:
Since you've used this same scope in here and in `app/helpers/topbar_helper.rb:9`, couldn't this be transformed into a scope in `Project`? maybe by the name `projects_joined_by_team` and passing the id of the team as a parameter. What do you think @talyssonoc?